subsys/bluetooth/shell/ticker.h

```diff
@@ -0,0 +1,18 @@
+/** @file
+ * @brief Bluetooth Controller's ticker functions
+ *
+ * This is not to be included by the application.
+ */
+
+/*
+ * Copyright (c) 2017 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __TICKER_H
+#define __TICKER_H
+
+int cmd_ticker_info(int argc, char *argv[]);
+
+#endif /* __TICKER_H */
```

Comment:
Since you decided to create your own shell module, it seems like this header file is unnecessary, as you only ever refer to the command callbacks from the same file that they're defined in.